src/components/product-filter/index.tsx

```diff
@@ -1,18 +1,50 @@
-import { forwardRef } from 'react';
+import { forwardRef, useState, useEffect } from 'react';
 import NiceSelect from '@ui/nice-select';
 import InputRange from '@ui/input-range';
 import { InputPrice } from '@types';
+import useAppSelector from '../../store/hooks/useAppSelector';
+import { selectState } from '../../features/leda-nft/store/leda-nft.slice';
 
-type Props = {
-  slectHandler: ({ value }: any, name: any) => void;
-  sortHandler: ({ value }: any, name: any) => void;
-  priceHandler: ({ value }: any, name: any) => void;
-  inputs: InputPrice;
-};
+/* type Props = {
```

Comment:
Please make sure to not leave commented code that we won't use. This makes the project dirtier. If you need the code somehow put a comment to keep it, otherwise remove it